control-plane/tests/test_service_persistence.py

```
@@ -4251,39 +4323,9 @@ def test_service_rejects_unknown_case_disposition(self) -> None:
 
     def test_service_rejects_raced_observation_identifier_collision(self) -> None:
         store, _ = make_store()
-        base_service = AegisOpsControlPlaneService(
-            RuntimeConfig(postgres_dsn="postgresql://control-plane.local/aegisops"),
-            store=store,
+        store, base_service, promoted_case, evidence_id, reviewed_at = (
```

Ruff (0.15.9)

[warning] Unpacked variable `base_service` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)